aws/logs_monitoring/lambda_function.py

```diff
@@ -1029,3 +1033,16 @@ def parse_service_arn(source, key, bucket, context):
                     region, accountID, clustername
                 )
     return
+
+
+def invoke_additional_target_lambdas(event):
+    lambda_client = boto3.client("lambda")
+    lambda_arns = DD_ADDITIONAL_TARGET_LAMBDAS.split(",")
+    lambda_payload = json.dumps(event)
+
+    for lambda_arn in lambda_arns:
+        lambda_client.invoke(
```

Comment:
One open question here is how we should handle errors here. Not super familiar with python pattern, so would love to get some feedback

Reply:
How about?

```python
for lambda_arn in lambda_arns:
    try:
        lambda_client.invoke(
            FunctionName=lambda_arn, InvocationType="Event", Payload=lambda_payload,
        )
    except Exception as e:
        logger.exception(f"Failed to invoke additional target lambda {lambda_arn} due to {e}")
```